css/global.less

```diff
@@ -21,6 +21,15 @@ body.custom-background-size {
     background-size:         cover !important;
 }
 
+/* Links */
+a {
+	text-decoration: none!important;
```

Comment:
This removes an accessibility feature for all links. With this in place the only thing that distinguishes links from other text is the colour. That means that people who cannot see colours very well won't be able to see what a link is.

Reply:
Didn't think of that one. For accessibility should I only use underline or can I use an alternate way (Besides color of course)? Like can I change the style of the underline?

Reply:
I removed the rule entirely

Reply:
You can generally use anything that is not purely based on colour. So, you can have a colour change if you also have something else. That doesn't need to be an underline. Only changing the background colour could also be okay in some circumstances (because background colour adds a "shape" if the colour difference is strong enough).